Plugins/Flow.Launcher.Plugin.Explorer/Views/ActionKeywordSetting.xaml.cs

```diff
@@ -63,11 +63,19 @@ private void OnDoneButtonClick(object sender, RoutedEventArgs e)
             }
 
 
-            if (CurrentActionKeyword.KeywordProperty == Settings.ActionKeyword.FileContentSearchActionKeyword 
-                && ActionKeyword == Query.GlobalPluginWildcardSign)
+            if (ActionKeyword == Query.GlobalPluginWildcardSign 
+                && (CurrentActionKeyword.KeywordProperty == Settings.ActionKeyword.FileContentSearchActionKeyword 
+                        || CurrentActionKeyword.KeywordProperty == Settings.ActionKeyword.QuickAccessActionKeyword))
```

Comment:
Why do we need this part of logic? The switch will handle the match

Reply:
yeah good pickup